src/Operators/areas_and_volumes.jl

```diff
@@ -0,0 +1,42 @@
+####
+#### Grid spacings
+####
+
+@inline Δx(i, j, k, grid) = grid.Δx
+@inline Δy(i, j, k, grid) = grid.Δy
+
+@inline ΔzC(i, j, k, grid::RegularCartesianGrid) = grid.Δz
+# @inline ΔzC(i, j, k, grid::VerticallyStretchedCartesianGrid) = @inbounds grid.ΔzC[k]
+
+@inline ΔzF(i, j, k, grid::RegularCartesianGrid) = grid.Δz
+# @inline ΔzF(i, j, k, grid::VerticallyStretchedCartesianGrid) = @inbounds grid.ΔzF[k]
+
+####
+#### Areas
+####
+
+@inline Ax_aac(i, j, k, grid) = Δy(i, j, k, grid) * ΔzF(i, j, k, grid)
+@inline Ax_aaf(i, j, k, grid) = Δy(i, j, k, grid) * ΔzC(i, j, k, grid)
+
+@inline Ay_aac(i, j, k, grid) = Δx(i, j, k, grid) * ΔzF(i, j, k, grid)
+@inline Ay_aaf(i, j, k, grid) = Δx(i, j, k, grid) * ΔzC(i, j, k, grid)
+
+@inline Az(i, j, k, grid) = Δx(i, j, k, grid) * Δy(i, j, k, grid)
+
+const AxC = Ax_aac
+const AxF = Ax_aaf
+const AyC = Ay_aac
+const AyF = Ay_aaf
+
+####
+#### Volumes
+####
+
+@inline V_aac(i, j, k, grid) = Δx(i, j, k, grid) * Δy(i, j, k, grid) * ΔzF(i, j, k, grid)
+@inline V_aaf(i, j, k, grid) = Δx(i, j, k, grid) * Δy(i, j, k, grid) * ΔzC(i, j, k, grid)
+
+const VC = V_aac
```

Comment:
Should we do this? This confused me below. Also I don't think `V_aac` makes sense. The volume has a 3D location.

Reply:
See my comments above.

Reply:
Got rid of `VC`.